common/src/types.rs

```diff
@@ -10,6 +10,20 @@ pub type ConsensusRound = u64;
 pub type FinalizationProof = Vec<TypedSignature<BlockHeader>>;
 pub type MemberName = String;
 
+#[derive(Debug, Serialize, Deserialize, Clone, Hash, Eq, PartialEq)]
+pub struct DelegationTransactionData {
+    pub delegator: PublicKey,
+    pub delegatee: PublicKey,
+    pub governance: bool,
+    pub timestamp: BlockHeight,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, Hash, Eq, PartialEq)]
+pub struct UndelegationTransactionData {
+    pub delegator: PublicKey,
+    pub timestamp: BlockHeight,
+}
+
```

Comment:
It seems like something went wrong when resolving conflicts. Please delete this.

Reply:
reverted.